### PR TITLE
install mergify

### PR DESCRIPTION
**Pull request by alessio**
______

For contributor use:

- [ ] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/gaia/blob/master/CONTRIBUTING.md#pr-targeting))
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/master/docs/building-modules/structure.md).
- [ ] Wrote unit and integration [tests](https://github.com/cosmos/gaia/blob/master/CONTRIBUTING.md#testing)
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)
- [ ] Added  relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [ ] Re-reviewed `Files changed` in the Github PR explorer

______

For admin use:

- [ ] Added appropriate labels to PR (ex. `WIP`, `R4R`, `docs`, etc)
- [ ] Reviewers assigned
- [ ] Squashed all commits, uses message "Merge pull request #XYZ: [title]" ([coding standards](https://github.com/tendermint/coding/blob/master/README.md#merging-a-pr))

### CODE REVIEW

File: .mergify.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+pull_request_rules:
+  - name: automerge to master with label automerge and branch protection passing
+    conditions:
+      - "#approved-reviews-by>1"
+      - base=master
+      - label=automerge
+    actions:
+      merge:
+        method: squash
+        strict: true
```
Comment (tac0turtle) [inline .mergify.yml:10]:
```suggestion
        strict: true
        commit_message: title+body
```

this takes in the title and description of prs. 

the only downside is all the check boxes that these templates have, but may be better to have a bot comment those check boxes

**Review by tac0turtle — APPROVED** [review]

File: .mergify.yml (hunk 1 above)
Comment (alessio) [reply .mergify.yml:10]: oh... taking in the checkboxes is not great

Comment (alessio) [reply .mergify.yml:10]: BTW I just copied the file as-is from cosmos-sdk

Comment (tac0turtle) [reply .mergify.yml:10]: yes yes not a issue, I may make a pr in a bit to change something that may make mergify better. We did it on tendermint and find it way better

**Review by fedekunze — APPROVED** [review]

**Pull request merged**